lib/common/http2client.c

```diff
@@ -104,6 +104,7 @@ struct st_h2o_http2client_stream_t {
     } input;
 
     int *notify_destroyed;
+    int _use_expect;
```

Comment:
```suggestion
    int use_expect;
```
Nitpick, but we do not need to have an underscore for fields declared in .c; they are inherently local to the file.